client/src/Tickets/TicketModal.jsx

```diff
@@ -52,19 +54,19 @@ function ClientDropMenu({ lookUp }) {
 }
 
 Charger.propTypes = {
-  stateChange: PropTypes.func.isRequired,
```

Comment:
Be observant to naming patterns in the existing libraries/code you're using... onClick, onChange, onSubmit... see the pattern? It's good practice to follow the pattern of naming conventions that exist already...